atm.js

```diff
@@ -1,3 +1,111 @@
-$(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+$("document").ready(function(){
+  console.log("hello");
+
+  //*******Checking Deposit START*********
+
+
+
+  $("#checking > .deposit").on("click", function(evt){
+    // prevent default action
+    evt.preventDefault();
+    // get user input && convert to float with 2 decimal placess
+    var newInput = $("#checking > .input").val()
+    var floatnewInput = parseFloat(newInput).toFixed(2);
+    console.log(floatnewInput);
+
+    // update total
+    //  get current total
+    var total = $("#checking > .balance").html()
+    var numerictotal = parseFloat(total.split("$")[1])
+    var actualtotal = parseFloat(floatnewInput) + numerictotal
+    if (actualtotal > 0) {
+      $("#checking > .balance").css("background-color", "#E3E3E3");
+    }
+    $("#checking > .balance").html("$ " + actualtotal.toFixed(2));
+    //clear field
+    $("#checking > .input").val("")
+
+
+
+  })
+
+  //*********Checking Deposit END************
+
+  //*********Savings Deposit START***********(change Variables)
```

Comment:
there seems to be a decent amount of repetition from this function to the other one happening with checking. Definitely some opportunities for refactoring.